### DISCUSSION TITLE
Kernel getting crashed while executing specific code blocks

### DISCUSSION THREAD

**Discussion by arjunsk07** [microsoft/vscode-jupyter#15588] [Questions and Answers]
Hello, 
I am working on vs code jupyter notebook, with a python environment kernel. 
It was all working fine till last week, and now when I run specific code blocks (related to training of a model), the kernel is getting crashed producing error like this:
![Screenshot from 2024-04-23 10-53-27](https://github.com/microsoft/vscode-jupyter/assets/93582711/a304f715-deca-4b5d-940f-eae9951c32c2)

Also when I check the log it is like this:
![image](https://github.com/microsoft/vscode-jupyter/assets/93582711/9b8afa80-f740-4bf4-8edc-7aa132bbb39f)

I also tried reducing the model complexity but still the same issue is coming up. I was able to run these code block without any issues with the same complexity till last week and it was not throwing any error at that time. Now I really don't know what is the reason behind this.Can anyone support me to solve this issue?

Some extra information, this is the kernel that I am using:
![image](https://github.com/microsoft/vscode-jupyter/assets/93582711/4ffe23dd-32cb-450c-8777-f6bec508b240)

this is the exact code block which is throwing the error:
![image](https://github.com/microsoft/vscode-jupyter/assets/93582711/bb384fc4-2848-49c2-a19d-806ca0d5675d)

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/15589
